src/pages/serverError/serverError.module.css

```diff
@@ -0,0 +1,129 @@
+@import "../../app/reset.css";
+@import "../../app/globals.css";
+
+html {
+	font-size: 100%;
+}
```

Comment:
in our project, rem is used instead of px for the css properties font-size and line-height. This thing will allow you to avoid bugs in cross-browser development
14px = 1.4rem
This is precisely why the font-size property of 62.5% is specified for the html tag in the reset.css file